src/core/settings/production.py

```diff
@@ -72,3 +72,5 @@
 # Sendinblue api and settings
 SIB_API_KEY = env('SIB_API_KEY')
 SIB_LIST_ID = env.int('SIB_LIST_ID')
+
+CORS_ALLOWED_ORIGINS = env('CORS_ALLOWED_ORIGINS')
```

Comment:
Peut-être garder `env.list` pour être cohérent (mais c'est du pinaillage) ?

```suggestion
CORS_ALLOWED_ORIGINS = env.list('CORS_ALLOWED_ORIGINS')
```